Add edge prior subsection and supplement (#17)
greenelab/xswap-manuscript: content/02.body.md

### Edge prior

We introduce the "edge prior" to quantify the probability that two nodes are connected based only on their degree.
The edge prior can be estimated using the maximum likelihood estimate for the binomial distribution success probability--the fraction of permuted networks in which a given edge exists.
Based only on permuted networks, the edge prior does not contain any information about the true edges in the (unpermuted) network.
The edge prior is a numerical feature that can be computed for each node pair in a network, and we compared its ability to predict edges in three tasks, discussed below.

### Edge prior approximation

We also considered the possibility that the probability of an edge existing across permuted networks could be written as a closed form equation involving the node pair's degree.

> Without the assumption of independent node pairs that we believe is incorrect for XSwap, we were unable to find a closed-form solution giving the edge prior.

Reviewer: ```suggestion
We were unable to find a closed-form solution giving the edge prior without assuming independent node pairs, which we believe is incorrect for XSwap.
```

Reviewer: I found this hard to parse with the other framing. I think this keeps the meaning and lets someone read it straight through.
